aperturedb/Images.py

```diff
@@ -110,6 +121,10 @@ def __init__(self, db, batch_size=100, response=None, **kwargs):
         self.bbox_label_prop = "_label"
         self.get_image = True
 
+        if response is not None:
+            self.images_ids = list(
+                map(lambda x: x[self.img_id_prop], response))
+
```

Comment:
it's hard to tell what is gong on here - what exactly is response in this context? I assume we are getting the image_ids at construction if a response is passed in - so it this allowing a quick construct of a Images with an already in-hand response?

Reply:
Yes, here is an [example](https://github.com/aperture-data/demos/blob/badf7f0d530b06321ac2f287392c5cfdab59c095/ml_notebooks/ROI-based%20Queries.ipynb) that would be useful to understand.